src/modules/feature-modules/assessment/pages/innovation/overview/overview.component.html

```diff
@@ -64,12 +64,58 @@ <h2 class="nhsuk-heading-m">Innovation summary</h2>
     </div>
   </dl>
 
-  <h2 class="nhsuk-heading-m">Innovator contact details</h2>
-  <dl class="nhsuk-summary-list">
-    <div *ngFor="let item of innovatorSummary" class="nhsuk-summary-list__row">
-      <dt class="nhsuk-summary-list__key">{{ item.label }}</dt>
-      <dd class="nhsuk-summary-list__value text-pre-wrap">{{ item.value }}</dd>
-    </div>
-  </dl>
+  <div class="full-width-section bg-color-white">
+    <h2 class="nhsuk-heading-m nhsuk-u-padding-top-5">Innovator details</h2>
+    <dl class="nhsuk-summary-list">
+      <div *ngFor="let item of innovatorSummary" class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">{{ item.label }}</dt>
+        <dd class="nhsuk-summary-list__value text-pre-wrap">{{ item.value }}</dd>
+      </div>
+    </dl>
+
+    <details class="nhsuk-details nhsuk-u-padding-top-2">
+      <summary class="nhsuk-details__summary" (click)="onShowCollaboratorsClick()">
+        <span class="nhsuk-details__summary-text">
+          Collaborators
+        </span>
+      </summary>
+      <div class="nhsuk-details__text">
+
+        <theme-spinner *ngIf="isCollaboratorsLoading; else collaboratorsLoaded"></theme-spinner>
+
+        <ng-template #collaboratorsLoaded>
+          
+          <p *ngIf=" showCollaboratorsHideStatus === 'opened' && innovationCollaborators.length === 0">There are no collaborators</p>
+
+          <ng-container *ngIf=" showCollaboratorsHideStatus === 'opened' && innovationCollaborators.length > 0">
+            <table class="nhsuk-table">
```

Comment:
Another simpler approach would be to load collaborators at the beginning, (even if after initial query's),

Thoughts?

Reply:
It does look like it is a simpler approach but doesn't it affect the overall performace of the page? I opted to do it like this to avoid always loading information that is not always going to be accessed